src/app/code/community/Omikron/Factfinder/Model/SdkClient/Client.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+include __DIR__ . DIRECTORY_SEPARATOR . '../../../../../../../../vendor/autoload.php';
+
+use Omikron\FactFinder\Communication\Client\ClientBuilder;
+use Omikron_Factfinder_Model_SdkClient_ClientBuilderConfigurator as ClientBuilderConfiguration;
+use Omikron_Factfinder_Model_SdkClient_Helper_Credentials as Credentials;
+use Psr\Http\Message\ResponseInterface;
+use Omikron\FactFinder\Communication\Client\ClientInterface;
+use Omikron_Factfinder_Model_Config_Auth as AuthConfig;
+use Omikron\FactFinder\Communication\ServerUrl;
+
+class Omikron_Factfinder_Model_SdkClient_Client
+{
+    private const GET = 'GET';
+    private const POST = 'POST';
+
+    /** @var \Omikron\FactFinder\Communication\Credentials */
+    private $credentials;
+    /** @var string */
+    private $serverUrl;
```

Comment:
formatting is off, we used to put empty lines between the property and docblock of next one